src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -117,28 +116,31 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
 import org.hamcrest.CoreMatchers;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.http.HttpStatus;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.util.ResourceUtils;
 
-@RunWith(SpringRunner.class)
+@ExtendWith({SpringExtension.class, TestJobWatcherExtension.class})
 @SpringBootTest
 @ActiveProfiles({"google", "integrationtest"})
 @AutoConfigureMockMvc
-@Category(Integration.class)
-public class AzureIntegrationTest extends UsersBase {
+@Tag(Integration.TAG)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class AzureIntegrationTest extends UsersBase {
```

Comment:
Probably not worth fixing here, but I think using inheritance for this this kind of code sharing (especially in a test class) is a poor design. Making `UsersBase` an injectable component with singleton scope would be a way to rewrite this and keep the same functionality. This could be done while keeping all current uses the same.